### PR TITLE
Add TCB 53 with Coginiti

### PR DESCRIPTION
**Pull request by mosabua**
Preview at https://deploy-preview-568--trino.netlify.app/episodes/53

### CODE REVIEW

**Comment by mullinsms** [comment]
Only small thing, we always write CoginitiScript pascal case

File: _episodes/53.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+AI-enabled enterprise data workspace that integrates modular development,
+version control, and data quality testing throughout the analytic development
+lifecycle.
+
+<a href="https://www.coginiti.co">
+  <img src="{{site.baseurl}}/assets/images/logos/coginiti-small.png">
+</a>
+
+With the support for Trino, Coginiti as a client tool provides access to all the
+configured catalogs in Trino. It enables data engineers and analyst to work
```
Comment (colebow) [inline _episodes/53.md:127]:
```suggestion
configured catalogs in Trino. It enables data engineers and analysts to work
```

File: _episodes/53.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+Coginiti delivers higher-quality analytics faster. Coginiti provides an
+AI-enabled enterprise data workspace that integrates modular development,
+version control, and data quality testing throughout the analytic development
+lifecycle.
+
+<a href="https://www.coginiti.co">
+  <img src="{{site.baseurl}}/assets/images/logos/coginiti-small.png">
+</a>
+
+With the support for Trino, Coginiti as a client tool provides access to all the
```
Comment (colebow) [inline _episodes/53.md:126]:
```suggestion
With support for Trino, Coginiti as a client tool provides access to all the
```

File: _episodes/53.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+configured catalogs in Trino. It enables data engineers and analyst to work
+together in a shared platform, reducing duplication in their work, and bringing
+"Don't repeat yourself (DRY)" to analysts.
+
+We talk about why Coginiti added [support for
+Trino](https://www.coginiti.co/databases/trino/). Coginiti is not compute
```
Comment (colebow) [inline _episodes/53.md:132]:
```suggestion
Trino](https://www.coginiti.co/databases/trino/). Coginiti is not a compute
```

File: _episodes/53.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+platform itself, but access to many platforms enables a data blender thinking.
```
Comment (colebow) [inline _episodes/53.md:133]: This sentence kinda just reads weird. Not totally sure what you mean by "A data blender thinking"

File: _episodes/53.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+## Rounding out
+
+Our line up for speakers and sessions for Trino Summit is nearly finalized. Join
+us on the 13th and 14th of December for the free, virtual event. Stay tuned for
+details about all the sessions soon, and in the meantime - [don't forget to
+register](https://www.starburst.io/info/trinosummit2023/?utm_source=trino&utm_medium=website&utm_campaign=NORAM-FY24-Q4-EV-Trino-Summit-2023&utm_content=tcb).
+
+Our [Trino SQL training series]({% post_url 2023-09-27-training-series %}) just
+had a successful third session, yesterday and you can check out all the material
```
Comment (colebow) [inline _episodes/53.md:154]:
```suggestion
had a successful third session yesterday, and you can check out all the material
```

File: _episodes/53.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+* Predicate pushdown for real and double columns in MongoDB.
+* Support for Iceberg REST catalog in the `register_table` and `unregister_table` procedures.
+* Support for BEARER authentication for Nessie catalog.
+
+[Trino 433](https://trino.io/docs/current/release/release-433.html)
+
+* Improved support for Hive schema evolution.
+* Add support for altering table comments in the Glue catalog.
+
+Also note that Trino 433 also includes documentation for `CREATE/DROP CATALOG`.
```
Comment (colebow) [inline _episodes/53.md:82]: Should we clarify when this was added? Cause this is saying we added documentation, but nothing about when it became possible.

**Review by colebow — APPROVED** [review]
Mostly looks good.

File: _episodes/53.md (hunk 6 above)
Comment (mosabua) [reply _episodes/53.md:82]: It came over a few releases and quite a while ago .. so I think we should leave it out and have people just use the latest...

**Pull request merged**